python/Ganga/GPIDev/Lib/Registry/RegistrySlice.py

```diff
@@ -72,10 +80,8 @@ def do_collective_operation(self, keep_going, method, *args, **kwds):
 
     def ids(self, minid=None, maxid=None):
         "Get the list of job ids. 'minid' and 'maxid' specify optional (inclusive) slice range."
-        if maxid is None:
-            maxid = sys.maxsize
-        if minid is None:
-            minid = 0
+        maxid = maxid if maxid not None else sys.maxsize
+        minid = minid if minid not None else 0
```

Comment:
I think if the expression gets this complicated (ignoring the syntax error) then it's not worth using a ternary operator.